### ISSUE TITLE
https_proxy and socks5 issue

### ISSUE THREAD

**Issue by aleliaert** [cloudfoundry-incubator/credhub-cli#29]
It looks like CLI version 1.5.1 should support socks5 proxying, but I'm having issues.  It appears to be treating `socks5` as a hostname, not a scheme:
```
curl -sS -L -o credhub.tar.gz https://github.com/cloudfoundry-incubator/credhub-cli/releases/download/1.5.1/credhub-linux-1.5.1.tgz
tar xfz credhub.tar.gz 
chmod +x credhub

https_proxy=socks5://127.0.0.1:55000 ./credhub api --server https://192.168.200.20
Error connecting to the targeted API: "Get https://192.168.200.20/info: proxyconnect tcp: dial tcp: lookup socks5 on 10.94.224.10:53: no such host". Please validate your target and retry your request.
```
I"m also getting a SEGV when doing `credhub --version`:
```
CLI Version: 1.5.1
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x68f0c6]

goroutine 1 [running]:
github.com/cloudfoundry-incubator/credhub-cli/credhub.(*CredHub).Request(0x0, 0x79954e, 0x3, 0x79bc5c, 0xc, 0xc42010e900, 0x0, 0x0, 0x10, 0x74ac00, ...)
	/tmp/build/8dc578f1/go/src/github.com/cloudfoundry-incubator/credhub-cli/credhub/request.go:20 +0x26
<snip>
```
Thanks!

**Comment by bgandon** [comment]
Hi, I just submitted PR #31 for the SEGV error.
Hoping this can help and get merged quickly!

**Comment by bgandon** [comment]
For the SOCKS support, it seems that rebuilding credhub CLI with a recent version of Go is enough.
I guess the Concourse build chain just needs to be updated in this regard.

**Comment by aleliaert** [comment]
Thanks @bgandon!  This isn't a blocker for my team so we'll wait until the next release naturally occurs.

**Comment by ebeer** [comment]
This is fixed in cli version 1.5.2
